include/ghostty.h

```diff
@@ -412,6 +412,7 @@ typedef enum {
   GHOSTTY_FULLSCREEN_NATIVE,
   GHOSTTY_FULLSCREEN_NON_NATIVE,
   GHOSTTY_FULLSCREEN_NON_NATIVE_VISIBLE_MENU,
+  GHOSTTY_FULLSCREEN_TITLED_NON_NATIVE_VISIBLE_MENU,
```

Comment:
I think this naming doesn't match the pattern of other variants. It should be `GHOSTTY_FULLSCREEN_NON_NATIVE_TITLED_VISIBLE_MENU` probably along with all the other enums and so on throughout this PR.